apps/prairielearn/src/pages/home/home.html.ts

```diff
@@ -51,12 +52,15 @@ export function Home({
     },
     options: {
       fullHeight: true,
+      paddingBottom: false,
+      marginTop: false,
     },
     content: html`
       <h1 class="visually-hidden">PrairieLearn Homepage</h1>
+      <div class="mb-3"></div>
```

Comment:
Nit, why this? (I think all this discussion goes away if we're willing to show the footer above the fold, which IMO is OK)

Reply:
The height of the content needs to be calculated as 100% of the page minus the navbar height. I'm open to you playing around with this, but this was the best solution I came up with without impacting other pages.